gateway/protocol.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/pokt-network/poktroll/pkg/polylog"
```

Comment:
1. Can you add a make target to show how to use this?
2. Can you update some documentation to show how to use this?